packages/style-kit/stories/button.stories.scss

```diff
@@ -12,6 +12,11 @@
     }
   }
 
+  button.icon-text {
+    color: black;
+    padding: 0px;
+  }
+
```

Comment:
Is this used?

Reply:
No it isn't I'll remove and make another PR